### ISSUE TITLE
Ratios need oob handling

### ISSUE THREAD

**Issue by tombishop1** [tombishop1/itraxBook#3]
Sometimes ratios create `Inf` values. These need dealing with before plotting, either using `na_if(Inf)` or some clever use of `scale_x_continuous(oob = scales::oob_censor_any())`.

**Comment by tombishop1** [comment]
OK, a trivial reproducible example is `CD166_19_xrf$Fe/CD166_19_xrf$Sc`. 

A suggested fix:

```
CD166_19_xrf %>%
  mutate(`Fe/Sc` = Fe/Sc) %>%
  mutate_if(is.numeric, list(~na_if(., Inf))) %>%
  pull(`Fe/Sc`)
```

**Comment by tombishop1** [comment]
Closed with de8090c
